pkg/trtl/trtl.go

```diff
@@ -42,6 +43,16 @@ const (
 // to look in that namespace only.
 func (h *TrtlService) Get(ctx context.Context, in *pb.GetRequest) (*pb.GetReply, error) {
 	var err error
+	var ns string
+
+	if in.Namespace == "" {
+		ns = "default"
+	} else {
+		ns = in.Namespace
+	}
```

Comment:
Didn't love having to add this to each of the RPC calls but I couldn't think of another way to get you the call data reliably disaggregated by namespaces the way we have the `default` behavior set up now.

Reply:
I know what you mean, if we change the name of the "default" namespace, in Honu, this will cause a minor problem that we'll have to deal with. 

One option is to get the namespace from Honu after we perform the db operation. Both `Put` and `Delete` return object meta which has the Namespace on it - so that should be fairly straight forward. In `Get` we're only doing `h.db.Object` if `in.Options.ReturnMeta` is true - but for this option we could always do `h.db.Object` so that we can get access to the meta data and therefore the namespace. We'll incur some small parsing cost from Honu, but it's really pretty insignificant with our recent changes. It will also simplify `Get` which might make it easier to maintain - e.g. you wouldn't need two separate metrics calls like right now.